spec/support/fixtures/shared_commands.rb

```diff
@@ -438,6 +438,18 @@ def call(**params)
     end
   end
 
+  class Namespace < Dry::CLI::Command
```

Comment:
Isn't bizarre to have a "command" that the uniq usage is to print top level usage? I am wondering if it should be a better idea to move the usage of top class as specific dsl. Like suggested here https://github.com/dry-rb/dry-cli/issues/96 ?

Reply:
It is. I tried to implement the suggestion from #96, but I couldn't figure out how to do it. Maybe we can create the `Dry::CLI::Namespace` class, so It will look less strange.

Reply:
That's a good idea.

Reply:
@gustavothecoder I like the idea of this being a specialised `Namespace` class. That way, when the class is used, it will be clearer that its purpose is to provide information about a grouping of subcommands only, and not its own distinct command behaviour.

I prefer this help text residing in a concrete class instead of being part of the top-level command registration API, since that feels more consistent with how the rest of the CLI construction is done: a class for each thing.

If you're still interested in working on this PR, I'd love it if you could make this adjustment! Let me know what you think.

Reply:
@timriley Done! I think it looks better using the `Namespace` class.